packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -130,6 +131,9 @@ export async function runBuildAndSubmitAsync(projectDir: string, flags: BuildFla
   }
 
   if (!flags.wait) {
+    if (flags.json) {
+      printJsonOnlyOutput(startedBuilds);
```

Comment:
```suggestion
      printJsonOnlyOutput(startedBuilds.map(buildInfo => buildInfo.build));
```
without `--no-wait`  we are printing list of build fragments, here you are also including build profile